code/src/reducers/todoStore.js

```diff
@@ -0,0 +1,48 @@
+import { createSlice } from '@reduxjs/toolkit';
+// Inital state
+const myInitialState = {
+	todos: [
+		{ id: 1, task: 'example task', dueDate: '2020-03-30', taskDone: false },
+		{ id: 2, task: 'cook dinner', dueDate: '2020-04-30', taskDone: false },
+		{ id: 3, task: 'buy birthday gift for John', dueDate: '2020-08-20', taskDone: false },
+		{ id: 4, task: 'changing tires', dueDate: '2020-01-30', taskDone: true }
+	]
+};
+// Todo Store
+export const todoStore = createSlice({
+	name: 'todoStore',
+	initialState: myInitialState,
+	reducers: {
+		// Possible actione
+		// New Task
+		createTask: (state, action) => {
+			let lastId = 0;
+			if (state.todos.length !== 0) {
+				lastId = state.todos[state.todos.length - 1].id;
+			}
+			state.todos.push({ id: lastId + 1, ...action.payload, taskDone: false });
```

Comment:
nice way of dealing with ids!
also, nice that you used the spread operator. `...`